requirements/test.txt

```diff
@@ -1,12 +1,14 @@
 flake8
-pep8-naming==0.5.0
+pep8-naming
 flake8-import-order
 flake8-print
 flake8-strict
-pytest==4.0.0
-pytest-cov==2.4.0
+pytest
+pytest-cov
 # web app tests
-sanic==20.12.3
+sanic<=20.12.4; python_version <= '3.6'
```

Comment:
Curious, how did you calculate this?

I am still developing my python knowledge.

Reply:
I don't know how @di calculated it, but looking at the git blame for the `setup.py` file of the Sanic project, it's possible to see that Python 3.6 support [was removed from Sanic on 2021 Feb 21](https://github.com/sanic-org/sanic/commit/37f0a510639f6acbea2f44ab59be2fc5c2939ceb).

That same commit adds the following line to the project README:

> Minimum Python version is 3.7. If you need Python 3.6 support, please use v20.12LTS.

As of the time of this pull request, `sanic==20.12.4` was the latest in the v20.12 line. Other micro versions have since been released in that LTS line, the latest as of today being `sanic==20.12.6`.

I suppose this means that an alternative would be to specify instead this line as:

```suggestion
sanic~=20.12; python_version <= '3.6'
```